<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>121</width>
      <height>402</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="tb_support">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>СИСТЕМА 
 ОБЕСПЕЧЕНИЯ</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_spareParts">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>ЗАПАСНЫЕ 
 ЧАСТИ</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_product">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>ИЗДЕЛИЯ</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_warehouse">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>СКЛАДЫ</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="table_groups">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>191</width>
      <height>471</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="l_group">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ГРУППЫ ЗАПАСНЫХ ЧАСТЕЙ</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_product">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_product">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ИЗДЕЛИЯ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_add_group">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>570</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ДОБАВИТЬ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_edit_group">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>570</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>РЕДАКТИРОВАТЬ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_delete_group">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>570</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>УДАЛИТЬ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ЗАПАСНЫЕ ЧАСТИ</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_spare_parts">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>461</width>
      <height>281</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pb_add_part">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>570</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ДОБАВИТЬ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_edit_part">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>570</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>РЕДАКТИРОВАТЬ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_delete_part">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>УДАЛИТЬ</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_supply_warehouses">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>НАЛИЧИЕ НА СКЛАДАХ</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_supply_warehouses">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>401</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tb_support</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onTBTabsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tb_spareParts</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onTBTabsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tb_product</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onTBTabsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tb_warehouse</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onTBTabsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_groups</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>onGroupTBClicked(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_add_group</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddGroupPbClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_delete_group</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDeleteGroupPbClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_add_part</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCreatePartPbClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_delete_part</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDeletePartPbClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_product</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onProductCbSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_spare_parts</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>onsparePartsTableSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_edit_group</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onEditGroupPbClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_edit_part</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>onEditPartPbClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTBTabsClicked()</slot>
  <slot>onGroupTBClicked(QTableWidgetItem*)</slot>
  <slot>onAddGroupPbClicked()</slot>
  <slot>onDeleteGroupPbClicked()</slot>
  <slot>onCreatePartPbClicked()</slot>
  <slot>onDeletePartPbClicked()</slot>
  <slot>onProductCbSelectionChanged()</slot>
  <slot>onsparePartsTableSelectionChanged()</slot>
  <slot>onEditGroupPbClicked()</slot>
  <slot>onEditPartPbClicked()</slot>
 </slots>
</ui>
